packages/terra-core-docs/src/terra-dev-site/doc/show-hide/About.1.doc.mdx

```diff
@@ -42,9 +45,120 @@ The Show-Hide component must be composed inside the [Base][1] component with a l
 ## Usage
 
 ```jsx
-import ShowHide from 'terra-show-hide';
+import ShowHide, { Paragraph } from 'terra-show-hide';
 ```
 
+#### A single paragraph of text
+  - At the top level of your component declare a `ref`, pass it as `focusRef` props to `ShowHide` component. Pass a preview node as a prop to `ShowHide` component.
+  - Use `Paragraph` component inside `<p>` tag. A `Paragraph` component allows `ShowHide` component to set focus on the beginning of the hidden portion of the text upon opening. 
+  - The text splitted in the preview and the rest of the text portions has to be passed as string props, and the `focusRef` should be forwarded to `Paragraph` component.
+
+  ```jsx
+  import React, { useRef, useState } from 'react';
+  import ShowHide, { Paragraph } from 'terra-show-hide';
+
+  const MyComponent = () => {
+    const focusRef = useRef(null);
+    const [isOpen, setIsOpen] = useState(false);
+    const toggleShowHide = () => setIsOpen(!isOpen);
+
+    // Split text in the preview and the rest of the text portions.
```

Comment:
@eawww I can reproduce that. It only happens in <li> example, the JAWS gets confused with the previous example if it is open, but for list example only. Please test the list example with the previous example close for now, I am working on figuring out what causes this bug and how to avoid it.

Reply:
Final fix for JAWS and VO confusion between multiple examples: https://github.com/cerner/terra-core/pull/3838/commits/ef2f17da26d15791ecac86c8dde5f0b911c768d4